cylc/flow/prerequisite.py

```diff
@@ -75,6 +79,10 @@ def __init__(self, point):
         # * `False` (prerequisite unsatisfied).
         self._all_satisfied = None
 
+    def __str__(self):
+        """Useful for debugging; could do with better formatting."""
+        return f"{self.point}: {self.satisfied}, {self.conditional_expression}"
```

Comment:
This is more suited to `__repr__`
```suggestion
    def __repr__(self):
        return (
            f"<{type(self).__name__} "
            f"{self.point}: {self.satisfied}, {self.conditional_expression}>"
        )
```